src/App.css

```diff
@@ -12,6 +16,12 @@ a {
   color: black;
```

Comment:
define colors into root at top or also you define all the colors into a single file and then access it into other files

Reply:
sure!